scripts/pipeline-DEPLOY.sh

```diff
@@ -340,6 +340,7 @@ else
   ibmcloud ks cluster-service-bind \
     --cluster "$PIPELINE_KUBERNETES_CLUSTER_NAME" \
     --namespace "$TARGET_NAMESPACE" \
+    --key "secure-file-storage-appid-acckey-${APPID_GUID}" \
```

Comment:
should the unbind be changed? https://github.com/IBM-Cloud/secure-file-storage/blob/bc7f9bbd1387ff8d94c2b020e034a5831756f250/scripts/pipeline-UNINSTALL.sh#L36

Reply:
@l2fprod No, the cluser-service-unbind does not have a key parameter. It unbinds the existing credentials.